internal/commands/push/command.go

```diff
@@ -468,6 +470,12 @@ func createNewApp(ui terminal.UI, realmClient realm.Client, appDirectory, groupI
 		}
 	}
 
+	if proceed, err := ui.Confirm("Are these settings correct?"); err != nil {
```

Comment:
to stay consistent with our previous messaging... lets make this:

`"Please confirm the new app details shown above"`